workspaces/adventure-pack/goodies/typescript/Math.gcd/index.test.ts

```diff
@@ -0,0 +1,40 @@
+import { describe, expect, it } from "@jest/globals";
+import "./index";
```

Comment:
nit: please add an empty line between the imports of external code and internal code